docs/reference/cluster_manifest.md

```diff
@@ -513,3 +513,33 @@ Those parameters are grouped under the `tls` top-level key.
   relative to the "/tls/", which is mount path of the tls secret.
   If `caSecretName` is defined, the ca.crt path is relative to "/tlsca/",
   otherwise to the same "/tls/".
+
+## Change data capture streams
+
+This sections enables change data capture (CDC) streams e.g. into Zalando’s
+distributed event broker [Nakadi](https://nakadi.io/). Parameters grouped
+under the `streams` top-level key will be used by the operator to create a
+CRD for Zalando's internal CDC operator named like the Postgres cluster.
+Each stream object can have the following properties:
+
+* **database**
+  Name of the database from where events will be published via Postgres'
+  logical decoding feature. The operator will take care of updating the
+  database configuration (setting `wal_level: logical`, creating logical
+  replication slots, using output plugin `wal2json` and creating a dedicated
+  replication user). Required.
+
+* **tables**
+  Defines a map of table names and event types. The CDC operator is following
+  the [outbox pattern](https://debezium.io/blog/2019/02/19/reliable-microservices-data-exchange-with-the-outbox-pattern/)
+  meaning changes are only consumed from an extra table that already has the
+  structure of the event in the target sink. The operator will assume that this
+  outbox table is called like `<table>_<event_type>_outbox`. Required.
```

Comment:
From reading this, I don't understand what exactly is done by this operator, or the CDC operator, and what is the responsibility of the application.
Does this create the outbox table on the DB managed with this? Does this set up some logic (per triggers?) to replicate data from the origin table into the outbox table, or is this the responsibility of the application? 
If the latter, what's the purpose of having the original table mentioned at all?

Reply:
It's confusing because it is not correct. Application-level terminology from the Outbox Pattern has been misinterpreted and applied to the the DB layer. From a Postgres CDC point of view, change events are emitted from a single table, there is no "extra" table. 

A builder persists a json event payload into a column _on any declared table_. Any change to a row on that table is captured via Logical Replication, with `after` state of the column eventually being emitted onto Nakadi via FES CDC.

Reply:
```suggestion
* **tables**
  Defines a map of (outbox) table names and event types. The CDC operator is following
  the [outbox pattern](https://debezium.io/blog/2019/02/19/reliable-microservices-data-exchange-with-the-outbox-pattern/).
  This means that the application will put events into a column in the outbox table
  in the structure of the target event type, and the CDC operator will capture them
  shortly after the transaction is committed. Required.
```
(Of course, the implementation also needs to change accordingly.)

I guess we need to define the column name too (either document it, if it's fixed, or have it be configurable here)?

Reply:
Reads much better. 
In the FES CRD, we have it convention-over-configuration for the column name:
```
    PostgresWalToGenericNakadiEventData:
      type: object
      properties:
        payloadColumn:
          type: string
          description: Name of the column holding the Json payload for the Nakadi event.
          example: my_business_event
          default: "payload"
```

Reply:
I see some confusion, but I already talked to both ACID/Nakadi to setup internal example repo, with Postgers clusters, Nakadi event type, I think this helps. We were too detached.